packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -151,6 +151,12 @@ export class DbSyncStakePoolProvider extends DbSyncProvider(RunnableModule) impl
         }
 
         return (options: QueryStakePoolsArgs) => this.#builder.queryPoolAPY(hashesIds, this.#epochLength, options);
+      case 'ros':
+        throw new ProviderError(
```

Comment:
Shouldn't this behave the same as `case 'apy'`?

Reply:
Actually yes, but does this worth?

Planning for a db-sync drop I spent no time on `DbSyncStakePoolProvider` (but the not implemented error).

Reply:
Alright let's not spend time on this